.lintr

```diff
@@ -1,5 +1,6 @@
 linters: linters_with_defaults(
   line_length_linter = line_length_linter(120),
   cyclocomp_linter = NULL,
-  object_usage_linter = NULL
+  object_usage_linter = NULL,
+  indentation_linter = NULL
```

Comment:
Why did you add this?

Reply:
The new indentation checks were ridiculous and I could not find a suitable option for [indentation_linter](https://lintr.r-lib.org/reference/indentation_linter.html):
```r
R/panel_group.R:29:10: style: [indentation_linter] Indentation should be 8 spaces but is 10 spaces.
          allTags()
        ~^
R/plot_with_settings.R:606:4: style: [indentation_linter] Indentation should be 14 spaces but is 4 spaces.
    is.null(brush$panelvar2) && is.character(original_panelvar2)) {
   ^~~~~~~~~~~
R/shiny_brush examples.R:4:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    ui = fluidPage(
  ~^
R/shiny_brush examples.R:5:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        sidebarLayout(
    ~~~^
R/shiny_brush examples.R:6:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            sidebarPanel(
      ~~~~~^
R/shiny_brush examples.R:7:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                actionButton("newplot", "New plot")
        ~~~~~~~^
R/shiny_brush examples.R:8:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            ),
      ~~~~~^
R/shiny_brush examples.R:10:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                plotOutput("plot")
        ~~~~~~~^
R/shiny_brush examples.R:11:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            )
      ~~~~~^
R/shiny_brush examples.R:12:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        )
    ~~~^
R/shiny_brush examples.R:13:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    ),
  ~^
R/shiny_brush examples.R:15:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        output$plot <- renderPlot({
    ~~~^
R/shiny_brush examples.R:16:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            input$newplot
      ~~~~~^
R/shiny_brush examples.R:20:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:21:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    }
  ~^
R/shiny_brush examples.R:27:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    ui = basicPage(
  ~^
R/shiny_brush examples.R:28:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        fluidRow(
    ~~~^
R/shiny_brush examples.R:29:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            column(
      ~~~~~^
R/shiny_brush examples.R:30:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                width = 4,
        ~~~~~~~^
R/shiny_brush examples.R:32:20: style: [indentation_linter] Indentation should be 10 spaces but is 20 spaces.
                    height = 300,
          ~~~~~~~~~^
R/shiny_brush examples.R:36:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                ),
        ~~~~~~~^
R/shiny_brush examples.R:41:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            ),
      ~~~~~^
R/shiny_brush examples.R:43:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                width = 4,
        ~~~~~~~^
R/shiny_brush examples.R:46:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            ),
      ~~~~~^
R/shiny_brush examples.R:48:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                width = 4,
        ~~~~~~~^
R/shiny_brush examples.R:51:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            )
      ~~~~~^
R/shiny_brush examples.R:52:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        )
    ~~~^
R/shiny_brush examples.R:53:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    ),
  ~^
R/shiny_brush examples.R:55:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        data <- reactive({
    ~~~^
R/shiny_brush examples.R:56:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            input$newplot
      ~~~~~^
R/shiny_brush examples.R:59:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:61:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            d <- data()
      ~~~~~^
R/shiny_brush examples.R:63:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:65:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            cat("Click:\n")
      ~~~~~^
R/shiny_brush examples.R:67:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:69:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            cat("Hover (throttled):\n")
      ~~~~~^
R/shiny_brush examples.R:71:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:73:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            cat("Brush (debounced):\n")
      ~~~~~^
R/shiny_brush examples.R:75:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:77:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            # For base graphics, we need to specify columns, though for ggplot2,
      ~~~~~^
R/shiny_brush examples.R:81:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                return()
        ~~~~~~~^
R/shiny_brush examples.R:82:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            }
      ~~~~~^
R/shiny_brush examples.R:84:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:86:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            res <- brushedPoints(data(), input$plot_brush, "speed", "dist")
      ~~~~~^
R/shiny_brush examples.R:88:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                return()
        ~~~~~~~^
R/shiny_brush examples.R:89:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            }
      ~~~~~^
R/shiny_brush examples.R:91:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:92:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    }
  ~^
R/shiny_brush examples.R:99:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    ui = basicPage(
  ~^
R/shiny_brush examples.R:100:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        fluidRow(
    ~~~^
R/shiny_brush examples.R:101:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            column(
      ~~~~~^
R/shiny_brush examples.R:102:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                width = 4,
        ~~~~~~~^
R/shiny_brush examples.R:104:20: style: [indentation_linter] Indentation should be 10 spaces but is 20 spaces.
                    height = 300,
          ~~~~~~~~~^
R/shiny_brush examples.R:107:24: style: [indentation_linter] Indentation should be 12 spaces but is 24 spaces.
                        id = "image_hover",
            ~~~~~~~~~~~^
R/shiny_brush examples.R:110:20: style: [indentation_linter] Indentation should be 10 spaces but is 20 spaces.
                    ),
          ~~~~~~~~~^
R/shiny_brush examples.R:112:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                )
        ~~~~~~~^
R/shiny_brush examples.R:113:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            ),
      ~~~~~^
R/shiny_brush examples.R:115:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                width = 4,
        ~~~~~~~^
R/shiny_brush examples.R:118:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            ),
      ~~~~~^
R/shiny_brush examples.R:120:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                width = 4,
        ~~~~~~~^
R/shiny_brush examples.R:123:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            )
      ~~~~~^
R/shiny_brush examples.R:124:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        )
    ~~~^
R/shiny_brush examples.R:125:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    ),
  ~^
R/shiny_brush examples.R:127:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        output$image <- renderImage({
    ~~~^
R/shiny_brush examples.R:128:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            input$newimage
      ~~~~~^
R/shiny_brush examples.R:130:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            # Get width and height of image output
      ~~~~~^
R/shiny_brush examples.R:134:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            # Write to a temporary PNG file
      ~~~~~^
R/shiny_brush examples.R:137:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            png(outfile, width = width, height = height)
      ~~~~~^
R/shiny_brush examples.R:141:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            # Return a list containing information about the image
      ~~~~~^
R/shiny_brush examples.R:143:16: style: [indentation_linter] Indentation should be 8 spaces but is 16 spaces.
                src = outfile,
        ~~~~~~~^
R/shiny_brush examples.R:148:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            )
      ~~~~~^
R/shiny_brush examples.R:149:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:151:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            cat("Click:\n")
      ~~~~~^
R/shiny_brush examples.R:153:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:155:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            cat("Hover (throttled):\n")
      ~~~~~^
R/shiny_brush examples.R:157:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:159:12: style: [indentation_linter] Indentation should be 6 spaces but is 12 spaces.
            cat("Brush (debounced):\n")
      ~~~~~^
R/shiny_brush examples.R:161:8: style: [indentation_linter] Indentation should be 4 spaces but is 8 spaces.
        })
    ~~~^
R/shiny_brush examples.R:162:4: style: [indentation_linter] Indentation should be 2 spaces but is 4 spaces.
    }
  ~^
tests/testthat/test-plot_with_settings.R:65:6: style: [indentation_linter] Indentation should be 12 spaces but is 6 spaces.
      ggplot2::geom_point())
     ^~~~~~~
```